modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,58 @@
+
+* With this update, you can manage resources with Argo CD that are also being used as template resources or "source" by an Operator. link:https://issues.redhat.com/browse/GITOPS-982[GITOPS-982]
+
+* With this update, the Operator will now configure the Argo CD workloads with the correct permissions to satisfy the Pod Security Admission that has been enabled for Kubernetes 1.24. link:https://issues.redhat.com/browse/GITOPS-2026[GITOPS-2026]
+
+* With this update, Config Management Plugins 2.0 is supported. The Argo CD custom resource now allows to specify sidebar containers for the repo server. link:https://issues.redhat.com/browse/GITOPS-766[GITOPS-776]
+
+* With this update, all communication between the Argo CD components and Redis cache are properly secured using modern TLS encryption. link:https://issues.redhat.com/browse/GITOPS-720[GITOPS-720]
+
+* This release of {gitops-title} adds support for IBM Z and IBM Power on {product-title} 4.10.
+
+[id="fixed-issues-1-6-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, `system:serviceaccount:argocd:gitops-argocd-application-controller` cannot create resource "prometheusrules" in API group `monitoring.coreos.com` in the namespace `webapps-dev`. This update fixes this issue and {gitops-title} is now able to manage all resources from the `monitoring.coreos.com` API group. link:https://issues.redhat.com/browse/GITOPS-1638[GITOPS-1638]
+
+* Before this update, While reconciling cluster permissions, every time the secret was getting deleted if the secret belonged to a cluster config instance. This update fixes this issue and now the `namespaces` field from the secret is deleted instead of deleting the complete secret. link:https://issues.redhat.com/browse/GITOPS-1777[GITOPS-1777]
+
+* Before this update, the Operator created the `Redis StatefulSet` with `podAffinity` rules, instead of `podAntiAffinity` rules while installing the HA variant of Argo CD through the Operator. This update fixes this issue and now the Operator creates the `Redis StatefulSet` with `podAntiAffinity` rules. link:https://issues.redhat.com/browse/GITOPS-1645[GITOPS-1645]
+
+* Before this update, Argo CD **ApplicationSet** had too many `ssh` Zombie processes. This update fixes this issue and now tini process reaper has been added into the **ApplicationSet** controller. This ensures, that SIGTERM is properly passed to the running process preventing it from being a Zombie process. link:https://issues.redhat.com/browse/GITOPS-2108[GITOPS-2108]
+
+[id="known-issues-1-6-0_{context}"]
+== Known issues
+
+*  {gitops-title} Operator can make use of RHSSO (KeyCloak) through OIDC in addition to Dex. However, with a recent security fix applied, the certificate of RHSSO cannot be validated in some scenarios. link:https://issues.redhat.com/browse/GITOPS-2212[GITOPS-2212]
```

Comment:
```suggestion
* {gitops-title} Operator can make use of RHSSO (KeyCloak) through OIDC in addition to Dex. However, with a recent security fix applied, the certificate of RHSSO cannot be validated in some scenarios. There is no workaround at this time. link:https://issues.redhat.com/browse/GITOPS-2212[GITOPS-2212]
```
- Mention whether there is a workaround or not per the known issue template here: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes